app/models/doctor.rb

```diff
@@ -0,0 +1,4 @@
+class Doctor < ApplicationRecord
+  has_many :reservations
```

Comment:
#Optional

Consider adding has_many through associations in both the Doctor and User models. This will enable accessing a doctor's patients through the Reservation join table and vice versa.